src/app.py

```
@@ -84,52 +159,63 @@ def _load_cached_resources():
+
+_chat_area()
 
 # 入力欄
 if question := st.chat_input("Obsidian ノートに質問する..."):
-    history = AppChatMessageHistory(st.session_state.session_id, max_turns=CHAT_HISTORY_TURNS)
+    session_id = st.session_state.session_id
+    shared = _get_shared_state()
+
+    logger.info("[%s] user submitted: %.60s", session_id[:8], question)
+
+    # 現在の会話履歴を取得（新しいユーザー質問は含まない）
```

Ruff (0.15.7)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)